control-plane/mbus-api/src/v0.rs

```diff
@@ -106,6 +106,8 @@ pub enum MessageIdVs {
     RemoveVolumeNexus,
     /// Generic JSON gRPC message
     JsonGrpc,
+    /// List of block devices
+    ListBlockDevices,
```

Comment:
To follow the previous ones, GetBlockDevices?

Reply:
Done